js/stanford_events_importer.js

```diff
@@ -231,37 +231,36 @@
 // -----------------------------------------------------------------------------
 
   if(typeof $.fn.quiet_highlight !== "function") {
-      /**
+    /**
      * A quick yellow highlight that fades out.
      */
     $.fn.quiet_highlight = function() {
```

Comment:
This was not for the timepicker specifically but rather for the auto-population of one hour ahead on the to date and time.